lib/drmhelper/tests/test_helper.py

```diff
@@ -130,55 +139,84 @@ def test_get_wvcdm_filename(self):
             result = h._get_wvcdm_filename()
             self.assertEqual(result, wvcdm_filename)
 
-    @mock.patch('drmhelper.utils.get_kodi_major_version')
-    def test_get_latest_ia_version(self, mock_kodi_maj_ver):
-        test_kodi_ver = 18
-        mock_kodi_maj_ver.return_value = test_kodi_ver
-        h = helper.DRMHelper()
-        result = h._get_latest_ia_version()
-        expected = config.LATEST_IA_VERSION[test_kodi_ver]['ver']
-        self.assertEqual(result, expected)
-
-    @mock.patch('drmhelper.utils.get_kodi_major_version')
-    def test_get_minimum_ia_version(self, mock_kodi_maj_ver):
-        test_kodi_ver = 18
-        mock_kodi_maj_ver.return_value = test_kodi_ver
-        h = helper.DRMHelper()
-        result = h._get_minimum_ia_version()
-        expected = config.MIN_IA_VERSION[test_kodi_ver]
-        self.assertEqual(result, expected)
-
-    def test_is_ia_current(self):
-        with mock.patch.object(helper.DRMHelper, '_get_minimum_ia_version',
-                               return_value='0.0.1'):
-            fake_addon = fakes.FakeAddon()
-            h = helper.DRMHelper()
-            result = h._is_ia_current(fake_addon)
-            self.assertTrue(result)
-
-    def test_is_ia_current_negative(self):
-        with mock.patch.object(helper.DRMHelper, '_get_minimum_ia_version',
-                               return_value='0.0.2'):
-            fake_addon = fakes.FakeAddon()
-            h = helper.DRMHelper()
-            result = h._is_ia_current(fake_addon)
-            self.assertFalse(result)
+    def test_get_wvcdm_paths(self):
+        for system in fakes.SYSTEMS:
+            rv = fakes.TRANSLATED_PATHS.get(system.get('system'))
+            with mock.patch.object(
+                    helper.DRMHelper, '_get_wvcdm_paths', return_value=rv):
+                fake_addon = fakes.FakeAddon()
+                h = helper.DRMHelper()
+                cdm_paths = h._get_wvcdm_paths(fake_addon)
+                self.assertEqual(rv[0], cdm_paths[0])
 
-    def test_is_ia_current_latest(self):
-        with mock.patch.object(helper.DRMHelper, '_get_latest_ia_version',
-                               return_value={'ver': '0.0.1'}):
-            fake_addon = fakes.FakeAddon()
-            h = helper.DRMHelper()
-            result = h._is_ia_current(fake_addon, latest=True)
-            self.assertTrue(result)
+    @responses.activate
+    def test_get_wvcdm_current_ver(self):
+        responses.add(responses.GET, config.CMD_CURRENT_VERSION_URL,
+                      body='1.4.9.1088')
```

Comment:
I'd actually be tempted here to use some dummy version number, so it's obvious that this is only for testing, like 9.9.9.9999